gcp_variant_transforms/testing/integration/vcf_to_bq_tests/presubmit_tests/medium_tests/test_annotation_pipeline_from_file_with_sharding_1_based.json

```diff
@@ -0,0 +1,96 @@
+[
+  {
+    "test_name": "test-annotation-pipeline-from-file-with-sharding",
+    "table_name": "test_annotation_pipeline_from_file_with_sharding",
```

Comment:
Shouldn't there be 1based in name here and for test name?

Reply:
Thanks for noticing this. I modified them.